globals.d.ts

```diff
@@ -1,5 +1,5 @@
-///<reference path="mozilla.d.ts" />
-///<reference path="modules/AuthVerifier.d.ts" />
+// <reference path="mozilla.d.ts" />
+// <reference path="modules/AuthVerifier.d.ts" />
```

Comment:
The triple dash is intended here, see also https://www.typescriptlang.org/docs/handbook/triple-slash-directives.html.

But unsure if they are actually doing anything relevant with us being a JS project with type checking only and not TypeScript.

Reply:
Restored the triple slashes and merged 2.x